test/DollarTest.cpp

```diff
@@ -0,0 +1,15 @@
+#include "gtest/gtest.h"
+
+#include "Dollar.h"
+
+using namespace testing;
+
+TEST(DollarTest, Multiplication){
+	
+	Dollar* five = new Dollar(5);
```

Comment:
I think that is this case you might want to avoid using pointers by declaring Dollar in the stack.